sklearn/multioutput.py

```diff
@@ -37,12 +39,90 @@ def _fit_estimator(estimator, X, y, sample_weight=None):
     return estimator
 
 
+def _partial_fit_estimator(estimator, X, y, classes=None, sample_weight=None,
+                           first_time=True):
+    if first_time:
+        estimator = clone(estimator)
+    else:
+        estimator = copy.copy(estimator)
+
+    if sample_weight is not None:
```

Comment:
Hi, thanks for looking into it. I understand that ``classes`` argument is not applicable for regressors. But in case someone inadvertently passes classes argument with a Regressor, the estimator would throw an error right ? Perhaps we can have separate ``partial_fit`` function in the sub classes ``MultiOutputRegressor`` and ``MultiOutputClassifier`` ? Just my 2c.

Also ``BaseSGD`` has an abstract definition for ``fit``, which is implemented in the subclasses ``BaseSGDClassifier`` and ``BaseSGDRegressor`` right ?

Please do let me know what you think ? Thanks.